### PR TITLE
Context wired up for Leaderboard and Alerts with CSS fixes

### PR DESCRIPTION
**Pull request by ashtilawat23**
Description
Wired up context for Alerts and Leaderboard. Data is coming in for both pages. Fixed CSS problems on Leaderboard.

What did you change?
Context added to both pages.

### CODE REVIEW

**Review by fidan93 — APPROVED** [review]
looks good 👍

File: src/components/pages/Leaderboard/RenderLeaderboard.js [hunk 1]
```diff
@@ -1,119 +1,49 @@
...
 import { LayoutContainer } from '../../common/';
 import NavBar from '../../common/NavBar';
 import './Leaderboard.css';
-import axios from 'axios';
 import Tabs from '../../common/Tabs';
 import { Layout } from 'antd';
 import NavMenu from '../../common/NavMenu';
+import { AdminContext } from '../../../state/contexts';
+import { getLeaderboard } from '../../../state/actions';
+import { ConsoleWriter } from 'istanbul-lib-report';
```
Comment (L-Steinmacher) [inline src/components/pages/Leaderboard/RenderLeaderboard.js:10]: What is this?

File: src/components/pages/Alerts/RenderAlerts.js [hunk 2]
```diff
@@ -1,25 +1,28 @@
...
-];
-
 function RenderAlerts(props) {
+  const context = useContext(AdminContext);
+
+  useEffect(() => {
+    if (context.clubs.length === 0) {
+      getClubs('authState', context);
+    }
+    if (context.memberReactions === 0) {
```
Comment (L-Steinmacher) [inline src/components/pages/Alerts/RenderAlerts.js:19]: add .length

File: src/components/pages/Alerts/RenderAlerts.js [hunk 3]
```diff
@@ -28,19 +31,19 @@ function RenderAlerts(props) {
...
-                  <div className="under-tabs-container">
-                    <div className="flags box">
-                      <h2>Negative Sentiment</h2>
-                    </div>
-                    <div className="insights box">
-                      <h2>Insights</h2>
-                    </div>
+          <Tabs>
+            {context.clubs.map(club => (
+              <div label={club.clubname}>
```
Comment (L-Steinmacher) [inline src/components/pages/Alerts/RenderAlerts.js:36]: Key may have to be on top level div in a map.

**Review by L-Steinmacher — CHANGES_REQUESTED** [review]
Add .length to line 19 in RenderAlerts.js as well as move the key from line 37 to line 36 in the same file.

File: src/components/pages/Leaderboard/RenderLeaderboard.js (hunk 1 above)
Comment (ashtilawat23) [reply src/components/pages/Leaderboard/RenderLeaderboard.js:10]: I removed it,

**Review by L-Steinmacher — APPROVED** [review]
Looks good and well organized.

**Pull request merged**